src/styles/globalStyle.js

```diff
@@ -1,14 +1,23 @@
 import { createGlobalStyle } from 'styled-components';
 
 const GlobalStyle = createGlobalStyle`
-  h1, h2, h3, h4, h5, h6 {
-    font-family: ${(props) => props.theme.fonts[0]};
-    color: ${(props) => props.theme.colors.textSecondary.main};
+  #root, html, body {
+    height: 100%;
```

Comment:
If you want the app to be at least a full window height (which is great) you can set `min-height: 100vh` and remove height here. Should fix the height issues in `HomePage` and `Header` as well